tcmur_cmd_handler.c

```diff
@@ -685,7 +685,7 @@ static int handle_unmap_in_writesame(struct tcmu_device *dev,
 	int ret;
 
 	/* If not aligned then falls back to the writesame without unmap */
-	if (lba % align || nlbas % align) {
+	if (align == 0 || lba % align || nlbas % align) {
```

Comment:
Do you know in which case will the `align` be `0` ? Could you enable the debug logs and reproduce it and then upload it ?

Thanks

Reply:
Okay, please ignore this. In the later comment you said you are using the `glfs`, not rbd.